### PR TITLE
fix: 🐛 missing binding for tvg_canvas_set_viewport c api

### PR DESCRIPTION
**Pull request by theashraf**
Related issue #2314

### CODE REVIEW

File: src/bindings/capi/tvgCapi.cpp [hunk 1]
```diff
@@ -138,6 +138,12 @@ TVG_API Tvg_Result tvg_canvas_sync(Tvg_Canvas* canvas)
     return (Tvg_Result) reinterpret_cast<Canvas*>(canvas)->sync();
 }
```
Comment (mgrudzinska) [inline src/bindings/capi/tvgCapi.cpp:140]: two new lines

**Review by mgrudzinska — APPROVED** [review]

File: src/bindings/capi/tvgCapi.cpp (hunk 1 above)
Comment (theashraf) [reply src/bindings/capi/tvgCapi.cpp:140]: @mgrudzinska updated

**Comment by theashraf** [comment]
@mgrudzinska it would be great if you can help me ship this fix to `0.13.x` branch, thanks

**Review by SergeyLebedkin — APPROVED** [review]

**Comment by mgrudzinska** [comment]
@theashraf done and thx :)

**Pull request merged**